src/System Application/App/Extension Management/src/ExtensionSettings.Page.al

```diff
@@ -97,6 +97,25 @@ page 2511 "Extension Settings"
                     ToolTip = 'Specifies if the publisher allows a symbol package download will contain the source code and all other resources that were part of the extension package.';
                 }
             }
+
+            group("Source Control details")
+            {
+                Caption = 'Source Control details';
```

Comment:
To me looks off with lowercase 'details'
All other good, however you decide

Reply:
I believe that this mix does not comply with the current styleguide:
https://learn.microsoft.com/en-us/style-guide/capitalization

Either `Source control details` or `Source Control Details`; same for the field group name.